src/main/java/com/airsquared/blobsaver/Controller.java

```diff
@@ -759,18 +824,47 @@ public void aboutMenuHandler() {
         }
     }
 
-    private void useMacOSMenuBar() {
+    private Stage createDefaultAboutStage() {
+        AboutStageBuilder stageBuilder = AboutStageBuilder.start(
+                labelMaker.getLabel(LabelName.ABOUT, "blobsaver"))
+                .withAppName("blobsaver").withCloseOnFocusLoss().withCopyright("Copyright \u00A9 " + Calendar
```

Comment:
The copyright year means when the project was worked on, not the current year.

Reply:
Removed.